src/schema/nodes/comment.ts

```diff
@@ -35,5 +39,8 @@ export const comment: NodeSpec = {
     contents: { default: '' },
     target: { default: '' },
     selector: { default: null },
+    resolved: { default: null },
+    contributions: { default: null },
+    originalText: { default: null },
```

Comment:
If it's possible it would be advisable to use default values corresponding to the actual type of the values. I'm guessing originalText would be empty string - '', resolved: false, contributions - empty array (not sure)